VDA-ERC20/contracts/VDA-V1.sol

```diff
@@ -99,9 +99,8 @@ contract VeridaToken is ERC20PausableUpgradeable, OwnableUpgradeable,
     /**
      * @dev Burn `amount` tokens from `to`.
      */
-    function burn(address from, uint256 amount) external override {
```

Comment:
@ITStar10 Why was the ability to burn tokens for a specific address removed?

Reply:
Just `burn` function was created by copy & paste the `mint(...)` function.
It's now updated to as following:
```
function burn(uint256 amount) external;
```
It's burning the token of function caller now.

Reply:
How would this be used?

Reply:
Any token holders can burn their token by this function.
Of course, normal users don't feel to burn their tokens.
By the way, token  manager - Verida need to burn tokens for several reasons.
If you think this is not necessary, we can remove this function.

Reply:
Once any token holders burn their tokens, the total supply of the Verida Token will be decreased.